base/db/man/build_insert_query.Rd

```diff
@@ -10,8 +10,6 @@ build_insert_query(values, table, .con)
 \item{values}{`data.frame` of values to write to SQL database}
 
 \item{table}{Name of target SQL table, as character}
-
-\item{.con}{[\code{DBIConnection}]:A DBI connection object obtained from \code{DBI::dbConnect()}.}
```

Comment:
I'm confused where this change comes from -- I don't see any corresponding changes to the PEcAn.DB source.